src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard/dashboard-v3.component.html

```diff
@@ -31,12 +31,15 @@
           </dd>
           <ng-container>
             <dt>Telemetry Dashboard
-              <span
-                class="badge"
-                [ngClass]="telemetryEnabled ? 'badge-success' : 'badge-secondary'"
-                [ngbTooltip]="getTelemetryText()" >
-                {{ telemetryEnabled ? 'Active' : 'Inactive' }}
-              </span>
+              <cds-tag *ngIf="telemetryEnabled === true"
+                       class="tags-success"
+                       [ngbTooltip]="getTelemetryText()">
+                Active
+              </cds-tag>
+              <cds-tag *ngIf="!telemetryEnabled"
+                       class="tags-secondary"
```

Comment:
Is not possible to use `[ngClass]`  the same way it was being used before, to conditionally apply one style or the other?

Reply:
Because cds-tag don't support ngclass

Reply:
For above screenshot unable to see data in services column unable. Will check and update service instances code as well

Reply:
![image](https://github.com/user-attachments/assets/78d171d8-6620-4904-8ce6-b9dc58ea7dc3)